src/tools/dev/masonry/opts/mb-3.2.1-darwin-10.14-x86_64-release.json

```diff
@@ -0,0 +1,54 @@
+{"bootstrap_visit":
+ {"version": "3.2.1",
+ "build_types": ["release"],
+ "branch": "3.2RC",
+ "arch":   "darwin-x86_64",
+ "cert":   "Developer ID Application: Lawrence Livermore National Laboratory (A827VH86QR)",
+ "entitlements": "/Users/harrison37/Work/github/visit-dav/3.2.0_release/visit/src/tools/dev/masonry/opts/visit.entitlements",
```

Comment:
Looks like you need a full path here for `entitlements` member. Any way it can be adjusted to use relative? Otherwise, the same `.json` file won't work on my system without editing.

Reply:
correct, I thought about making that change but the iteration time was too long for me to tackle it during the release process. Maybe we can create a ticket to look at this during the next release?

Reply:
Ok, I've added #17097.

Will I at least get an error that it can't find the file so I'll know to go fix that if I have to?